docs/gatsby-config.js

```diff
@@ -11,6 +13,15 @@ module.exports = {
         repoRootPath: '..',
       },
     },
+    {
+      resolve: `gatsby-plugin-alias-imports`,
+      options: {
+        alias: {
+          "@primer/components": process.env.NODE_ENV === 'production' ? 'node_modules/@primer/components' : path.resolve(__dirname, '../src')
```

Comment:
It might be nice to leave a comment here explaining why the alias has to be different in production vs development.

Reply:
but what if i don't know why 😂  jk i will add it

Reply:
Also, will this work with deploy previews? For example, if I make a change to a component, will that change take affect in the deploy preview for that pull?

Reply:
`/* DON'T EVEN THINK ABOUT TOUCHING THIS */`

Reply:
@colebemis probably not 😭 I didn't consider that 🤔

Reply:
Can it resolve to the dist folder in the production case?

Reply:
Nevermind, I just read your description where you said you tried this. 🤷‍♂️

Reply:
Also, I noticed with this approach that some of the styles are wonky - check out the link colors for an example. Maybe it has something to do with this error in the console?

```
It looks like there are several instances of 'styled-components' initialized in this application. This may cause dynamic styles not rendering properly, errors happening during rehydration process and makes your application bigger without a good reason.

See https://s-c.sh/2BAXzed for more info.
```

I don't get that error if I don't alias the @primer/components to the src folder.

Reply:
looks like I can fix that for the development environment at least by adding an alias for ```styled-components```
```
'styled-components': path.resolve(__dirname, '../node_modules/styled-components')
```